src/app/projects/projects.scss

```
@@ -0,0 +1,16 @@
+body {
+  .projects-container {
+    padding: 64px 0;
+  }
+  .card {
```

Can we put spaces between these selectors?

```suggestion
body {
  .projects-container {
    padding: 64px 0;
  }

  .card {
    height: 100%;
    display: flex;
    flex-direction: column;
  }

  .card-media {
    padding: 56.25% 0 0 0;
  }
  
  .card-content {
    flex-grow: 1;
  }
}
```

---

Same with all scss files/selectors.

---

Okay, I will do that.